tests/cli/test_deploy.py

```diff
@@ -4929,18 +4872,7 @@ def test_deploy_existing_deployment_with_no_changes_does_not_prompt_save(self):
 
         invoke_and_assert(
             command="deploy -n existing-deployment --cron '* * * * *'",
-            user_input=(
-                # decline remote storage
-                "n"
-                + readchar.key.ENTER
-                # accept create work pool
-                + readchar.key.ENTER
-                # choose process work pool
-                + readchar.key.ENTER
-                # enter work pool name
-                + "inflatable"
-                + readchar.key.ENTER
-            ),
+            prompts_and_responses=[("Would you like to save configuration", "y")],
```

Comment:
This is an example of what i'd consider a false negative, and one of the issues with our current pattern. This command didn't actually result in asking the user about remote storage or work pools. Previously the input was just ignored, but with the `prompts_and_responses` the test would have failed.